.github/workflows/lte-integ-test-magma-deb.yml

```diff
@@ -32,7 +32,7 @@ jobs:
         with:
           path: ~/.vagrant.d/boxes/magmacore-VAGRANTSLASH-magma_test
           key: vagrant-box-magma-test
```

Comment:
Why is this test box not set to `vagrant-box-magma-test-v1.2.20221012`?

Reply:
That's a bug, thank you. I mistakenly changed the line below that. Fixed